maxMoviesInYear.js

```diff
@@ -0,0 +1,50 @@
+const mockData= require('./mockResult.js')
+
+function findMaxMoviesInYear () {
+    const moviesInAYear = {};
+    for ( let movie of mockData ) {
+        movie.license.reduce( ( acc, currentValue,currentIndex ) => {  
+            let currentYear = currentValue;
+
+            // loop through the license array till the end of the array and not consider the second index of the license array since it would be duplicate entry.
+
+            while ( currentYear <= movie.license[movie.license.length - 1] && (currentIndex!==movie.license.length-1) ) {  
```

Comment:
Dangerous logic. Please improve

Reply:
I removed my reduce method and just implemented a simple while loop in my current version. Since using reduce was not being much of a help there because I had to put additional check on top of it.